File: pset6/mario.py
from cs50 import *
while (1):
    print("height: ",end="")
    n=get_int()
    n=int(n)
    if(n>=0 and n<24)  : 
        break
m=n-1;
for i in range(n):
        for j in range(n):
            if(j<m):
                print(" ",end="")
            else:
                print("#",end="")
        m=m-1
        print()

File: pset1/mario.c
#include<cs50.h>
#include<stdio.h>
int main(void)
{    int n,i,j,m;
    do{
     printf("height: ");
     n = get_int();
        
    }while(n<=0 || n>=24);
          m=n-1;
           for(i=0;i<n;i++)
           {
               for(j=0;j<=n;j++)
               {
                   if(j<m)
                    printf(" ");
                   else
                    printf("#");
               }
               m--;
               printf("\n");
           }
     
}